src/components/contestants/project_card.less

```diff
@@ -0,0 +1,94 @@
+@import "../../main";
+
+
+.projects-grid {
+   -moz-column-width: 15em;
```

Comment:
Pune link la [asta](http://stackoverflow.com/questions/22148063/making-a-pinterest-like-grid-with-bootstrap-the-boxes-are-not-displayed-right-b). Să nu îți fie frică niciodată să spui că ai luat de undeva. :smile: